docs/platforms/flutter/session-replay/index.mdx

```diff
@@ -59,12 +59,26 @@ Sampling allows you to control how much of your website's traffic will result in
    up to a minute of events prior to the error and continue recording until the session
    ends.
 
-Sampling starts as soon as a session begins. The <PlatformIdentifier name="replays-session-sample-rate" /> is then evaluated. If the session is sampled, replay recording will start immediately. If not, <PlatformIdentifier name="replays-on-error-sample-rate" /> will be evaluated. If the session is sampled at this point, the replay will be buffered and will only be uploaded to Sentry if an error occurs. 
+Sampling starts as soon as a session begins. The <PlatformIdentifier name="replays-session-sample-rate" /> is then evaluated. If the session is sampled, replay recording will start immediately. If not, <PlatformIdentifier name="replays-on-error-sample-rate" /> will be evaluated. If the session is sampled at this point, the replay will be buffered and will only be uploaded to Sentry if an error occurs.
 
 ## Privacy
 
-The SDK is recording and aggressively redacting (masking) all text and images, according to the configuration in `options.experimental.replay`.
-You can tune this and add custom masking rules to fit your needs. For example, you can explicitly mask or unmask widgets by type,
+The SDK is recording and aggressively redacting (masking) all `Text`, `EditableText`, and `Image` widgets.
+Masking in the Sentry Flutter SDK is based on Widget *types*, e.g. `Image`, not the string representation of the type (i.e. we check whether
+a `widgetInstance` should be masked by checking `if (widgetInstance is Image)` instead of `if (widgetInstance.runtimeType == 'Image')`).
```

Comment:
Is a best effort `string` based approach useful at all? For release builds specially, if it gives false sense of security because works in debug mode but not in release, it'd be far from idea

Reply:
> Is a best effort string based approach useful at all?

I don't think it is, that's why we don't have that - we only check by the actual type. Does the sentence need to be updated to make it clearer?

I have [an open PR](https://github.com/getsentry/sentry-dart/pull/2375) to error out when we encounter something that looks like it should be masked and it isn't (and it's not explicitly allowed either) but I wouldn't even mention that in the docs.